controllers/docker/test_docker_controller.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+import responses
```

Comment:
What's the latest one? This file or `tests/test_docker_controller.py`? Perhaps you committed both due to a move to a `tests/` folder?